microsoft/testsuites/hpc/infinibandsuite.py

```diff
@@ -360,10 +366,13 @@ def verify_open_mpi(self, environment: Environment, log: Logger) -> None:
             raise SkippedException(err)
 
         run_in_parallel([server_ib.install_open_mpi, client_ib.install_open_mpi])
-
         server_node.execute("ldconfig", sudo=True)
         client_node.execute("ldconfig", sudo=True)
 
+        # Only for mariner, we need to build intel benchmarking tools 
+        # as they are not included in our packages
+        server_ib.install_intel_mpi_benchmarking_tool()
```

Comment:
Please also add `if` block here to skip other distros. So the logic here is easier to know at the test case level.

Reply:
Sure.